samples/subsys/usb/testusb/README.rst

```diff
@@ -0,0 +1,58 @@
+.. _testusb-app:
+
+Testusb application sample
+##########################
+
+The testusb implements a loopback function. This function can be used
```

Comment:
`The testusb sample...`

Reply:
The sample itself does not contain anything, this is done in the USB device stack.

Reply:
Then who implements a loopback function?